Pod/Classes/CDTQQueryValidator.m

```diff
@@ -287,28 +349,46 @@ + (BOOL)validateClause:(NSDictionary *)clause
     //$exits lt
 
     NSArray *validOperators =
-        @[ @"$eq", @"$lt", @"$gt", @"$exists", @"$not", @"$ne", @"$gte", @"$lte" ];
+        @[ @"$eq", @"$lt", @"$gt", @"$exists", @"$not", @"$gte", @"$lte", @"$in" ];
```

Comment:
Maybe not that things like `$ne` are normalised before we get here, with a reference to the method where you define the mapping from $ne to $eq as it's negated pair.